### DISCUSSION TITLE
Passwordless login

### DISCUSSION THREAD

**Discussion by darkLord19** [mattermost/community#116] [Moonshot Ideas]
- User can request to get a magic link using which they can login without needing to provide password
- Link can be only used once
- Link should expire after predefined time

**Comment by chetanyakan** [comment]
Also, QR code scan login like discord

**Reply by Willyfrog** [reply]
this would be so cool to have to login with mobile!

**Comment by esethna** [comment]
Totally agree, our login flow sucks is so painful, especially on mobile. FaceID/biometrics is another option there

**Comment by cnu1812** [comment]
FaceId/biometric is quite better..!
